src/batch/src/executor/order_by.rs

```diff
@@ -252,7 +252,8 @@ impl OrderByExecutor {
                             Interval,
                             NaiveDate,
                             NaiveTime,
-                            NaiveDateTime
+                            NaiveDateTime,
+                            Struct
```

Comment:
Seems `List` has similar issue.

Reply:
It would better to have a tool that can enumerate all cases. Seems we are working on sqlsmith to do the job.